<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SignUpPage</class>
 <widget class="QWidget" name="SignUpPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="usernameLabel">
     <property name="text">
      <string>Username</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="passwordLabel">
     <property name="text">
      <string>Password</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="confirmPasswordLabel">
     <property name="text">
      <string>Confirm Password</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="usernameField"/>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="passwordField"/>
   </item>
   <item row="2" column="1">
    <widget class="QLineEdit" name="confirmPasswordField"/>
   </item>
   <item row="7" column="1">
    <widget class="QPushButton" name="finishButton">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Finish</string>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="lastNameLabel">
     <property name="text">
      <string>Last Name</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="firstNameLabel">
     <property name="text">
      <string>First Name</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QLineEdit" name="firstNameField"/>
   </item>
   <item row="4" column="1">
    <widget class="QLineEdit" name="lastNameField"/>
   </item>
   <item row="5" column="1">
    <widget class="QLabel" name="errorLabel">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
